src/mlpack/methods/ann/layer/concat.hpp

```diff
@@ -53,6 +53,19 @@ class Concat
   Concat(const bool model = false,
          const bool run = true);
 
+  /**
+   * Create the Concat object using the specified parameters.
+   *
+   * @param inputSize A vector denoting input size of each layer added.
+   * @param axis Concat axis.
+   * @param model Expose all network modules.
+   * @param run Call the Forward/Backward method before the output is merged.
+   */
+  Concat(arma::Row<size_t> inputSize,
+         const int axis,
```

Comment:
For consistency with the rest of the codebase I would like to use `size_t` instead of `int`, and for `-1` we could introduce a boolean `hasAxis`. Let me know what you think.

Reply:
Done